### DISCUSSION TITLE
How to account to account for concentration values in MoBi

### DISCUSSION THREAD

**Discussion by alexisdiaz7** [Open-Systems-Pharmacology/Forum#1323] [Questions & Problems]
I developed a post-partum adult female model in PK-Sims and then transferred it to MoBi to observe the effects of different medications during lactation. Is there a way to account for different concentrations of a drug both in plasma and milk within MoBi, or is this only possible to add in PK-Sims?

**Comment by msevestre** [comment]
@alexisdiaz7 Whatever you can do in PK-Sim, you can do in MoBi
What do you mean exactly by`account for different concentrations`?  You want to use the dynamic concentration in your equations?

**Reply by alexisdiaz7** [reply]
I am looking to include the milk concentrations in both milk and plasma in my model but am unsure if they would be added as "Observed Data" or directly into an equation.

**Comment by tobiasK2001** [comment]
Hi, this looks like an interesting project. Did you see that: https://www.page-meeting.org/default.asp?abstract=10113
Maybe the authors can give you some tips.

Best, Tobias

**Comment by StephanSchaller** [comment]
Hi @alexisdiaz7 , have a look at this publication: https://pubmed.ncbi.nlm.nih.gov/19710640/
This was done in PK-Sim in an earlier version but should give you a good idea about how to approach this.

Good Luck!
